build/js/live-editor.shared.js

```diff
@@ -136,6 +136,10 @@ window.ScratchpadRecord = Backbone.Model.extend({
         // Array of command arrays for each chunk. Only holds chunks that
         // have been saved.
         this.allSavedCommands = [];
```

Comment:
Build files were left out of the last PR, which is why these changes are here. I've sent a separate PR to Scott for them though, will merge that first.